src/core/api/clock.ts

```diff
@@ -34,6 +34,18 @@ import {
 import config from "../../config";
 import { getLeftSizeOfRange, getRange } from "../../utils/ranges";
 
+export type IMediaInfosState =
+  "init" |
+  "canplay" |
+  "play" |
+  "progress" |
+  "seeking" |
+  "seeked" |
+  "loadedmetadata" |
+  "canplay" |
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Remove this duplicated type or replace with another one. ([typescript:S4621](https://sonarcloud.io/organizations/rx-player/rules#rule_key=typescript%3AS4621))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=rx-player&pullRequest=319&issues=AWYbSeM-vOWp7Yd636Jf&open=AWYbSeM-vOWp7Yd636Jf)

[ISSUE_KEY]: AWYbSeM-vOWp7Yd636Jf